docs/HowToBuildFromSource.md

```diff
@@ -2,21 +2,21 @@
 
 If you want to use the latest version of ONE-vscode, let's do the following:
 
-## STEP1. Install vsce
+## STEP1. Clone repository
 
-[vsce](https://code.visualstudio.com/api/working-with-extensions/publishing-extension) is `Visual Studio Code Extension`.
-- You should install [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) before installing vsce
+git clone `ONE-vscode`
 
 ```console
-$ npm install -g vsce
```

Comment:
Thanks. I see the same warning that vsce is deprecated and renamed to @vscode/vsce 😄 
```
npm WARN deprecated vsce@2.15.0: vsce has been renamed to @vscode/vsce. Install using @vscode/vsce instead.
```

Reply:
Um, to generate vsix file, we need `vsce` package. That's why we need this code. How about changing from `vsce` to `@vscode/vsce`?

Reply:
@jyoungyun @vscode/vsce is already included in devDependency in package.json. So there's no need to guide it separately :-D